pyiron/gaussian/gaussian.py

```diff
@@ -2,14 +2,16 @@
 # Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
 # Distributed under the terms of "New BSD License", see the LICENSE file.
 
-import os
+import os,subprocess
 import numpy as np
 
 from pyiron.base.generic.parameters import GenericParameters
 from pyiron.atomistics.structure.atoms import Atoms
-from pyiron.atomistics.job.atomistic import AtomisticGenericJob
+from pyiron.atomistics.job.atomistic import AtomisticGenericJob,GenericOutput,Trajectory
 from molmod.io.fchk import FCHKFile
-from molmod.units import amu
+from molmod.units import amu,angstrom,electronvolt
+from molmod.periodic import periodic
+import tamkin
```

Comment:
`tamkin` is currently not part of the conda-forge channel, so if we want to use it I recommend adding it to conda-forge as well.

Reply:
Good idea, the tamkin version that I import here is a reduced python 3 version that can be found on my github.

Reply:
I started the submission to conda forge https://github.com/conda-forge/staged-recipes/pull/9429